sklearn/mixture/tests/test_gmm.py

```diff
@@ -111,7 +121,11 @@ def test_lmvnpdf_full():
     reference = _naive_lmvnpdf_diag(X, mu, cv)
```

Comment:
that's not what the test tries to do. The change should be below in not using fullcv